projects/u-mill/white_mesa/decantation_filtration.py

```diff
@@ -485,20 +495,6 @@ def __init__(self):
         # S T A T E  P H A S E
         #***************************************************************************************
 
-        # Single Tank Decantation State
```

Comment:
Why are you removing the state phase that keeps track of the volume of liquid in the STD?

Reply:
This must have reverted back by accident. I had to redownload the entire repository due to the extent of conflicts in my version of the code. This can be put back. Why are we tracking this?